### PR TITLE
Radio button markup standardization

### PR DESCRIPTION
**Pull request by hbillings**
So it turns out that the extra space was a literal extra space and had nothing to do with markup, but this still standardizes the markup if we want to do that. _shrugging man made of symbols_

### CODE REVIEW

File: frontend/source/sass/data_explorer.scss [hunk 1]
```diff
@@ -534,7 +534,7 @@ a.focus-input {
 #query-types {
   padding: 0 1rem;
   label {
-    display: inline-block;
+    //display: inline-block;
```
Comment (toolness) [inline frontend/source/sass/data_explorer.scss]: Do you think we should remove this comment entirely?

**Comment by toolness** [comment]
Oops, it looks like this change breaks accessibility actually, which is why the CircleCI build is failing: [Radio inputs with the same name attribute value must be part of a group](https://dequeuniversity.com/rules/axe/2.1/radiogroup).

File: frontend/source/sass/data_explorer.scss (hunk 1 above)
Comment (hbillings) [reply frontend/source/sass/data_explorer.scss]: oh hmm...yeah, that looks like cruft from me trying to figure out what was goin' on.

**Comment by hbillings** [comment]
Ohhh, interesting! I took off the `role="group"` designation when I changed the radios' parent from a `div` to a `ul`. Will fix (and, man, I wish some of these things had more clear names...)

**Comment by ericronne** [comment]
@hbillings this still in need of a merge?
(i just merged `1759-explorer-search-box-improvements` into dev)

**Comment by hbillings** [comment]
Yeah, but I was hoping you'd verify it looked right in your branch first.

**Comment by ericronne** [comment]
sorry, was unaware of this pr until now

**Comment by hbillings** [comment]
@ericronne Could you still check out this branch and review the way it looks?

**Comment by ericronne** [comment]
wait, i *was* aware of this branch! 
at one point in time, many moons ago (ie yesterday).
👴 

looks like the conversion to `<li>` added some vertical space, i'll adjust, hold pls...

**Pull request merged**